zzk/service/hoststate_test.go

```diff
@@ -33,27 +33,23 @@ var (
 	ErrTestNoAttach = errors.New("could not attach to container")
 )
 
+const (
+	servicePath string = "/services/serviceid"
+	hostPath    string = "/hosts/hostid"
+	hostId      string = "hostid"
+	serviceId   string = "serviceid"
+	serviceName string = "serviceA"
+	imageId     string = "imageid"
+	containerId string = "containerid"
+)
+
```

Comment:
I disagree. Glenn actually recommended putting the strings in constants, and I agree with his recommendation. There's already one test case that requires a different value, and I just used a different constant there. Using constants makes it clear where values are supposed to be the same (rather than just coincidentally agreeing), and prevents weird failures due to typos. Also, if we want to change the test values for some reason, we can simply change the value in one place, rather than having to pick through the source code deciding whether each instance of a string should be changed or not.